04.ls/ls.rb

```diff
@@ -20,6 +53,77 @@ def prepare_files(options)
   options['r'] ? sorted_files.reverse : sorted_files
 end
 
+def prepare_file_details(files)
+  file_details = []
+  blocks = 0
+
+  files.each do |filename|
+    filestat = File.stat(filename)
+    blocks += filestat.blocks
+    file_details << build_file_detail(filename, filestat)
+  end
+  [blocks, file_details]
+end
+
+def build_file_detail(filename, filestat)
+  file_permissions = build_file_permissions(filestat)
```

Comment:
```suggestion
  permissions = build_permissions(filestat)
```

file_ はなくてもいいかも
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name

Reply:
file_を削除し、変数名をスッキリさせました。